### PR TITLE
requested tweaks, part 2

### PR DESCRIPTION
**Pull request by ericfershtman**
## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by molly-moen — APPROVED** [review]

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -2534,10 +2534,10 @@
...
   tutorial_rpstress_longdescription: "Use Scratch to create a stress ball that you can squash!"
-  tutorial_bsdcause_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Support My Cause focuses on designing and coding a landing page that will collect data from supporters of your favourite cause."
+  tutorial_bsdcause_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Support My Cause focuses on designing and coding a landing page that will collect data from supporters of your favourite cause. #codeisyourvoice"
```
Comment (clareconstantine) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Are we intentionally spelling "favourite" the British way?

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -2534,10 +2534,10 @@
...
   tutorial_vidcodeanimoji_longdescription: "Use a loop to iterate over an array, causing an emoji to animate through multiple options."
   tutorial_vidcodeanimoji_longdescription: "Use a loop to iterate over an array, causing an emoji to animate through multiple options."
-  tutorial_bsdinspire_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The People Who Inspire Me focuses on creating a webpage to showcase the top 3 people who have been an inspiration to you. "
+  tutorial_bsdinspire_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The People Who Inspire Me focuses on creating a webpage to showcase the top 3 people who have been an inspiration to you. #codeisyourvoice"
```
Comment (clareconstantine) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Adding a missing space

```suggestion
  tutorial_bsdinspire_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. The People Who Inspire Me focuses on creating a webpage to showcase the top 3 people who have been an inspiration to you. #codeisyourvoice"
```

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 3]
```diff
@@ -2552,7 +2552,7 @@
...
-  tutorial_bsdtrivia_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Trivia Game Maker focuses on building an interactive trivia game where you also get to come up with the trivia questions."
+  tutorial_bsdtrivia_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Trivia Game Maker focuses on building an interactive trivia game where you also get to come up with the trivia questions. #codeisyourvoice"
```
Comment (clareconstantine) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Adding missing spaces
```suggestion
  tutorial_bsdtrivia_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. The Trivia Game Maker focuses on building an interactive trivia game where you also get to come up with the trivia questions. #codeisyourvoice"
```

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 4]
```diff
@@ -2561,7 +2561,7 @@
...
   tutorial_codespeakmeal_longdescription: "Learn to code an app that helps users make healthy food decisions."
   tutorial_itchfish_longdescription: "Scratch 3.0 tutorial for making a game with a shark chasing and eating fish disrupted when a diver is eaten accidentally."
-  tutorial_bsdwebsite_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. 'The My Favourites Website focuses on creating a webpage of the top 3 things you are interested in or love doing. "
+  tutorial_bsdwebsite_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. 'The My Favourites Website focuses on creating a webpage of the top 3 things you are interested in or love doing. #codeisyourvoice"
```
Comment (clareconstantine) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Adding missing spaces
```suggestion
  tutorial_bsdwebsite_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. 'The My Favourites Website focuses on creating a webpage of the top 3 things you are interested in or love doing. #codeisyourvoice"
```

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 5]
```diff
@@ -2579,7 +2579,7 @@
...
   tutorial_cuttle_longdescription: "Challenge yourself with these fun and interactive tasks and learn all about computational thinking."
-  tutorial_bsdjokes_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Jokes and Riddles focuses on creating a set of interactive cards that contain your favorite jokes and riddles and the answers! "
+  tutorial_bsdjokes_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Jokes and Riddles focuses on creating a set of interactive cards that contain your favorite jokes and riddles and the answers! #codeisyourvoice"
```
Comment (clareconstantine) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Adding missing spaces
```suggestion
  tutorial_bsdjokes_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. The Jokes and Riddles focuses on creating a set of interactive cards that contain your favorite jokes and riddles and the answers! #codeisyourvoice"
```

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 1 above)
Comment (clareconstantine) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Adding missing spaces
```suggestion
  tutorial_bsdcause_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. The Support My Cause focuses on designing and coding a landing page that will collect data from supporters of your favourite cause. #codeisyourvoice"
```

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 6]
```diff
@@ -2534,10 +2534,10 @@
...
   tutorial_rpstress_longdescription: "Use Scratch to create a stress ball that you can squash!"
-  tutorial_bsdcause_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Support My Cause focuses on designing and coding a landing page that will collect data from supporters of your favourite cause."
+  tutorial_bsdcause_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. The Support My Cause focuses on designing and coding a landing page that will collect data from supporters of your favourite cause. #codeisyourvoice"
```
Comment (clareconstantine) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]:
```suggestion
  tutorial_bsdcause_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. The Support My Cause focuses on designing and coding a landing page that will collect data from supporters of your favorite cause. #codeisyourvoice"
```

**Review by clareconstantine — APPROVED** [review]

**Pull request merged**